src/components/OpenWith/ExecuteForm.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @flow
+ * @file Form to invoke an integration via POST
+ * @author Box
+ */
+
+import * as React from 'react';
+
+type Props = {
+    formData: ExecuteAPI,
+    id: string,
+};
+
+const ExecuteForm = ({ formData: { url, params }, id }: Props) => (
+    <React.Fragment>
+        <form
+            id={`bcow-execute-form-${id}`}
```

Comment:
Is ID needed? Can we use React.createRef() and pass back reference

Reply:
yeah that sounds better

Reply:
React states that passing up the reference isn't ideal. Daniel had the nice suggestion to do the submit in `componentDidMount` of the form component itself.